### DISCUSSION TITLE
Remove specific text from a node

### DISCUSSION THREAD

**Discussion by alirezahmz** [remirror/remirror#1807] [Q&A]
I have a menu and I want it to work like a Notion's menu, what I do and expect:     
I write a text, then I hit the slash and search for one of the menu items as shown below, and after clicking on H1, I expect that only the "/heading" text will be removed from the entire text.      
To do this, I used useSuggest and I have the text range, but I don't know how to delete the text and update it.    
I couldn't find anything about this in the documentation and examples.      
For the sake of Jesus, please check it out faster.    
     
![slashMenu](https://user-images.githubusercontent.com/64639961/181770072-1f3924ea-0a73-4114-9518-aafa9901b7b6.PNG)

**Comment by 47vigen** [comment]
you can use chained commands 

```
const chain = useChainedCommands();
chain.delete({ from: number, to: number }).run();
```
check doc here : https://remirror.io/docs/hooks/use-chained-commands/
